tests/de/lock_HassTurnOff.yaml

```diff
@@ -0,0 +1,13 @@
+language: de
+tests:
```

Comment:
please also add tests for area related sentences

Reply:
added